### PR TITLE
Modified index.md

### PR DESCRIPTION
**Pull request by kanishka-sahoo**
Made a few changes regarding pesu timetable and exams.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -85,12 +84,12 @@ There are three types of examinations/tests held in PES:
 Normal assignments of around 20 marks, conducted in the class and written in blue books. 1 assignment is conducted for every unit, and the aggregate marks are considered for the final score.
 
 !!!info 
-There are no lab exams for Physics or Chemistry. Only for C and Python.
+There are no lab exams for Physics or Chemistry. Only for C and Python. In Physics, however, there may be a 30-mark lab MCQ.
```
Comment (silicoflare) [inline index.md:88]: Isn't the 30-marks lab MCQ a part of the ESA??

File: index.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ As long as you're in the first year, you are a part of the Science and Humanitie
 ***
 
 ### THE COURSE CONTENT
-Students are divided into Physics and Chemistry cycles, usually based on alphabetical order of their names. But there are some common subjects in each semester irrespective of the cycle. If you get the Physics cycle in Sem 1, you'll get Chemistry cycle in Sem 2 and vice versa.
+Students are divided into Physics and Chemistry cycles, usually based on alphabetical order of their names. But there are some common subjects in each semester irrespective of the cycle. If you get the Physics cycle in Sem 1, you'll get Chemistry cycle in Sem 2 and vice versa. The final letter determines the sem: A for odd sems and B for even sems.
```
Comment (silicoflare) [inline index.md]: Don't worry about this change, I'll make a separate section in the article about course codes.

**Review by silicoflare — CHANGES_REQUESTED** [review]
Please go through the comments and make the necessary changes

**Comment by silicoflare** [comment]
@kanishka-sahoo Please make the specified changes

File: index.md (hunk 1 above)
Comment (kanishka-sahoo) [reply index.md:88]: Yeah it is, i guess. But it's based on the lab component and added to the theory final scores.

**Comment by silicoflare** [comment]
Did you pull from the main branch recently, coz I made many changes.

**Review by silicoflare — APPROVED** [review]

**Pull request merged**